website/docs/tutorials/guides/deploying.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: Deploying (1 min)
+---
+
+# Deploying
+
+Protostar supports deploying smart contracts to a given network with the `protostar deploy` command. It has a similar interface to the `starknet deploy` command. [Read the CLI reference for the deploy command](/docs/cli-reference#deploy) to learn more about all supported arguments.
+
+## Example — deploying the default contract
+After [compiling your project](/docs/tutorials/guides/compiling), you can deploy the contract in the following way.
+
+```
+$ protostar deploy ./build/main.json --network alpha-goerli
+```
+  
+ ```shell title="Deployment output"
+[INFO] Deploy transaction was sent.
+Contract address: 0x06a5ea9e42c921bd58e24b8da9d1fc91a488df0700b173f1c6bb0e453f68afec
```

Comment:
Maybe we should put mock values here 😛 ? Idk what is deployed there though just raising it as a random thought

Reply:
IMO it doesn't  matter